src/gui/watchedfolderoptionsdialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>462</width>
-    <height>325</height>
+    <height>364</height>
```

Comment:
revert it?

Reply:
>revert it?

New widgets were added thus height is increased.